HPCC-14024 DOCS:Target Name to Scope

g-pan opened the pull request:
FIX HPCC-14024 DOCS:Target Name to Scope
Change all references from Name Prefix
to Target Scope - and corresponding Images.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@JamesDeFabia please review

g-pan committed:
HPCC-14024 DOCS:Target Name to Scope

FIX HPCC-14024 DOCS:Target Name to Scope
Change all references from Name Prefix
to Target Scope - and corresponding Images.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
All looks good except one additional change is needed:
IMDB\IMDB.xml Line 427: <para>Complete the Name prefix ...

g-pan committed:
HPCC-14024 DOCS:Target Name to Scope

FIX HPCC-14024 DOCS:Target Name to Scope
Change all references from Name Prefix
to Target Scope - and corresponding Images.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
Good to merge now
